techniques/general/G84.html

```diff
@@ -12,108 +11,88 @@
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content that collects user input where a limited set of values must be input.</p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p>限定された値の入力のみを受け付けるコンテンツ</p>
+            <p>これは、次の達成基準に関連する達成方法である:</p>
             <ul>
-               <li><a href="https://w3c.github.io/wcag/understanding/error-identification">Success Criterion 3.3.1: Error Identification</a> (Sufficient)
+               <li><a href="https://w3c.github.io/wcag/understanding/error-identification">達成基準 3.3.1: エラーの特定</a> (十分な達成方法)
                </li>
-               <li><a href="https://w3c.github.io/wcag/understanding/error-suggestion">Success Criterion 3.3.3: Error Suggestion</a> (Sufficient)
+               <li><a href="https://w3c.github.io/wcag/understanding/error-suggestion">達成基準 3.3.3: エラー修正の提案</a> (十分な達成方法)
                </li>
-               <li><a href="https://w3c.github.io/wcag/understanding/status-messages">Success Criterion 4.1.3: Status Messages</a> (Sufficient as a way to meet <a href="../aria/ARIA19">ARIA19: Using ARIA role=alert or Live Regions to Identify Errors</a>)
+               <li><a href="https://w3c.github.io/wcag/understanding/status-messages">達成基準 4.1.3: ステータスメッセージを理解する</a> (<a href="../aria/ARIA19">ARIA19: エラーを特定するために、ARIA role=alert 又はライブリージョン (Live Regions) を使用する</a>の達成方法として十分)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>When users enter input that is validated, and errors are detected, the nature of the
-               error needs to be described to the user in manner they can access. One approach is
-               to present an alert dialog that describes fields with errors when the user attempts
-               to submit the form. Another approach, if validation is done by the server, is to return
-               the form (with the user's data still in the fields) and a text description at the
-               top of the page that indicates the fact that there was a validation problem, describes
-               the nature of the problem, and provides ways to locate the field(s) with a problem
-               easily. The "in text" portion of the Success Criterion underscores that it is not
-               sufficient simply to indicate that a field has an error by putting an asterisk on
-               its label or turning the label red. A text description of the problem should be provided.
+            <h2>解説</h2>
+            <p>利用者が入力したものがバリデートされ、エラーが検出された場合、利用者がアクセスできる方法でそのエラーの性質を説明する必要がある。一つのアプローチは、利用者がフォームを送信しようとするときに、エラーのあるフィールドを説明したアラートダイアログを提示するものである。バリデーションがサーバーサイドで行われる場合のもう一つのアプローチは、(利用者のデータがフィールドに残っている) フォームと、ページの上部にバリデーションで問題があったことを示すテキストの説明を返し、その問題の性質を説明して、問題のあるフィールドを簡単に見つけられる方法を提供するものである。達成基準の「テキストで」の部分は、フィールドのラベルにアスタリスクを付けたり、ラベルのテキストを赤色に変えたりしてエラーがあることを示すだけでは十分でないことを強調している。問題のテキストによる説明を提供すべきである。
```

Comment:
もう一つのアプローチの箇所、若干語順を変えました（1つ前の文とあわせるイメージで）

```suggestion
            <p>利用者が入力したものがバリデートされ、エラーが検出された場合、利用者がアクセスできる方法でそのエラーの性質を説明する必要がある。一つのアプローチは、利用者がフォームを送信しようとするときに、エラーのあるフィールドを説明したアラートダイアログを提示するものである。もう一つのアプローチは、バリデーションがサーバーサイドで行われる場合、(利用者のデータがフィールドに残っている) フォーム及び、ページの上部にバリデーションで問題があったことを示すテキストの説明を返し、その問題の性質を説明して、問題のあるフィールドを簡単に見つけられる方法を提供するものである。達成基準の「テキストで」の部分は、フィールドのラベルにアスタリスクを付けたり、ラベルのテキストを赤色に変えたりしてエラーがあることを示すだけでは十分でないことを強調している。問題のテキストによる説明を提供すべきである。
```

Reply:
語順の変更、問題ありません。もんどさん修正案のほうが、「一つのアプローチは〜。もう一つのアプローチは〜」と対称関係が明確になるので、かえって読みやすいですね。